test/e2e/framework/util.go

```diff
@@ -876,9 +876,21 @@ func (f *Framework) MatchContainerOutput(
 	Logf("Trying to get logs from node %s pod %s container %s: %v",
 		podStatus.Spec.NodeName, podStatus.Name, containerName, err)
 
-	// Sometimes the actual containers take a second to get started, try to get logs for 60s
-	logs, err := e2epod.GetPodLogs(f.ClientSet, ns, podStatus.Name, containerName)
+	// Sometimes the actual containers take a few seconds to get started, try to get logs for 1m.
+	var logs string
+	var podLogErr error
+	err = wait.PollImmediate(
+		2*time.Second, /* from pods/wait.go:poll */
+		1*time.Minute,
+		func() (bool, error) {
+			logs, podLogErr = e2epod.GetPodLogs(f.ClientSet, ns, podStatus.Name, containerName)
+			if err == nil {
+				return true, nil
+			}
+			return false, nil
+		})
 	if err != nil {
+		err = fmt.Errorf("%w: %w", err, podLogErr)
```

Comment:
Need to fix here due to the error:
```
test/e2e/framework/util.go:893:9: fmt.Errorf call has more than one error-wrapping directive %w
```
as https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/109578/pull-kubernetes-verify/1516918382241255424

Reply:
thanks, fixed.